cdci_data_analysis/analysis/parameters.py

```diff
@@ -437,6 +446,8 @@ def value(self, v):
         else:
             self._v=None
 
+    def get_value_in_default_format(self, value):
+        return int(value)
 
     @staticmethod
     def check_int_value(value, units=None,name=None):
```

Comment:
yeah, this function is not really checking for int. It accepts float, at least. And it does it in a peculiar way...

Reply:
actually we could use `check_int_value` (and `check_float_value`) for `Float`

Reply:
why use `check_int_value` for `float`? We really just care that value can be cast to `float`. `int` can be safely cast. Not the opposite though: from float to int.

Reply:
I meant that we could use use `check_int_value` in this case (`Integer`), and `check_float_value` for `Float`.

Or I misunderstood you comment?

Reply:
we could, but we need to adapt them.

right now, they are identical:

https://github.com/oda-hub/dispatcher-app/blob/d8b313db28c2de51fd90e4640064f2183a9437a1/cdci_data_analysis/analysis/parameters.py#L459-L474

https://github.com/oda-hub/dispatcher-app/blob/d8b313db28c2de51fd90e4640064f2183a9437a1/cdci_data_analysis/analysis/parameters.py#L405-L421

Reply:
the only difference is the conversion to a `np.float` / `np.int` object, hence my question: is that really necessary?

Reply:
it is really necessary to rewrite them according to our expectations!

Reply:
ok some differentiation from the methods can be made

but my question is specific to the creation of a numpy object (`np.float(value)`/`np.int(value)`) are those necessary? why they were there in first place?

Reply:
and BTW I also discovered that the usage of `np.float` and `np.int` is deprecated

[https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations](https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations)

and that `float` and `int` respectively can be used

Reply:
No idea. It's an old part and not written by me. Some workaround for something.

Reply:
It will no longer use `np.float` or `np.int`, rather `float` or `int`